### DISCUSSION TITLE
'STM32F103C8T6' not work PWM Function.

### DISCUSSION THREAD

**Discussion by SeongJongKwak** [stm32duino/Arduino_Core_STM32#2046] [General]
https://www.stm32duino.com/viewtopic.php?p=12515#p12515

https://img1.daumcdn.net/thumb/R1280x0/?scode=mtistory2&fname=https%3A%2F%2Fblog.kakaocdn.net%2Fdn%2Fw79AA%2FbtslcDQ49OX%2FKcQDAkiKcKHLArhccLERMk%2Fimg.jpg

Only 10 out of 20 'PWM PIN' works.
What's the reason?
I wonder. I can't understand.

**Comment by fpistm** [comment]
You do not use this core but Roger's one.
https://github.com/rogerclarkmelbourne/Arduino_STM32
